src/main/webapp/script.js

```diff
@@ -102,10 +102,16 @@ function setUpGraphs() {
       getOutdatedSDK(jobData);
       sdkVisited = true;
     }
-    google.charts.setOnLoadCallback(getTotalCosts(jobData));
-    google.charts.setOnLoadCallback(getAverageCosts(jobData));
+    google.charts.setOnLoadCallback(
+      drawPieChart(getTotalCosts(jobData), 'Total Cost of Jobs Per Category', 'totalCost-container')
+    );
+    google.charts.setOnLoadCallback(
+      drawPieChart(getAverageCosts(jobData), 'Average Cost of Jobs Per Category', 'averageCost-container')
+    );
     google.charts.setOnLoadCallback(getDailyView(jobData));
-    google.charts.setOnLoadCallback(getFailedJobs(jobData));
+    google.charts.setOnLoadCallback(
+      drawPieChart(getFailedJobs(jobData), 'Total Number of Failed Jobs Per Category', 'failedJobs-container')
+    );
```

Comment:
Maybe you could have a function like this
`function setGraphOnLoad(data, string) {  
  google.charts.setOnLoadCallback(  
      drawPieChart(data, string);
    );
}`

You would have your code more modularized.

Reply:
Good idea :D Changed in 93be99b.